### DISCUSSION TITLE
Formik `TypeError: Cannot read properties of undefined (reading 'getFieldProps')` when migrating from v6 to v8

### DISCUSSION THREAD

**Discussion by lainercoder** [storybookjs/storybook#28092] [Help]
### Summary

Hi! We're in the process of migrating from Storybook v6.5 to Storybook v8.1. We're loving all the updates & enhancements in the new version!

We have some components that use Formik v2. They were displaying properly in our old version of storybook. However, in the v8 storybook, they display correctly locally, but crashed on published versions with the following error: `TypeError: Cannot read properties of undefined (reading 'getFieldProps')`

This is how each of the stories looks:
```tsx
export const DefaultStory: StoryObj<typeof meta> = {
    args: {},
    render: ({ name, ...args }) => (
        <Formik
            enableReinitialize
            initialValues={{
                [name]: '',
            }}
            onSubmit={() => {}}
            validationSchema={object().shape({
                [name]: date(),
            })}
        >
            <Form>
                <FormikDatePicker {...args} name={name} />
            </Form>
        </Formik>
    ),
};
```

Appreciate any insight or guidance you have!

**Comment by Ashroy30** [comment]
Hey, did you ever figure this out? getting the same error :/
